README.md

```diff
@@ -36,6 +36,10 @@ dependencies {
 runIdeForUiTests {
     systemProperty "robot-server.port", System.getProperty("robot-server.port")
 }
+
+maven {
```

Comment:
if you want to copy the build.gradle structure, then this needs to be in a `repository` block